testing/scripts/Makefile

```diff
@@ -57,7 +57,6 @@ kind_build_images: build_protos kind_build_engine kind_build_operator kind_build
 
 .PHONY: helm_setup
 helm_setup:
-	helm repo add stable https://kubernetes-charts.storage.googleapis.com/
```

Comment:
So we don't use any of the stable charts and can just remove?

Reply:
From what I have seen it seems that helm-3 may have everything we use by default, but I'm running the tests to confirm, namely as otherwise we'll have to add `helm repo add stable https://charts.helm.sh/stable`, as that seems to be the recommended way as opposed to that one.